tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -250,6 +250,16 @@ internal static partial class ConfigurationKeys
         /// </summary>
         public const string DogStatsDArgs = "DD_DOGSTATSD_ARGS";
 
+        /// <summary>
+        /// Used to force the loader to start the tracer agent (in case automatic instrumentation is disabled)
+        /// </summary>
+        public const string AasEnableCustomTracing = "DD_AAS_ENABLE_CUSTOM_TRACING";
+
+        /// <summary>
+        /// Used to force the loader to start dogstatsd (in case automatic instrumentation is disabled)
+        /// </summary>
+        public const string AasEnableCustomMetrics = "DD_AAS_ENABLE_CUSTOM_METRICS";
+
```

Comment:
Ah, here they are. Personally I'd put them in `AzureAppServices` with the other AAS variables, but either way, we should use these rather than the `private const`

Reply:
Done in bcb18c1aed9e291edfea674fe0f92e5d082659da